Dockerfile.tmpl

```diff
@@ -528,6 +528,9 @@ RUN mkdir -p /root/.EasyOCR/model && \
     rm /root/.EasyOCR/model/craft_mlt_25k.zip && \
     /tmp/clean-layer.sh
 
+# Add 7z support
+RUN apt-get install -y p7zip-full
```

Comment:
The compressed package will be part of that Docker image layer which will make the Docker image size larger make sure to add the following:

```
RUN apt-get install -y p7zip-full && \
    /tmp/clean-layer.sh
```

Alternatively (better), you can add it at line to line 62 next to the the `unzip` package.

Reply:
I've moved the install to line 62 as you suggested.